cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -115,7 +113,7 @@ class GetAllWork : public PooledWork<GfErrType>,
   TcrMessage* getReply() { return m_reply; }
 
   void init() {}
-  GfErrType execute(void) {
+  GfErrType execute() {
```

Comment:
Curious what was the warning/error message here?  At least in C, it is legal to declare a function like so

```c
void f()
{
}
```

And then call it like so...
```c
f(1, 2, 3);
```

Explicitly calling out void in the function arguments prevents that.  Perhaps C++ takes away the foot gun..

Reply:
Clang-tidy in CLion said it was `Redundant void argument list in function definition`.  My understanding was C++ didn't allow the C example you shared.  I didn't clean up the header file, it still has `(void)` with the `Redundant void argument` message

Reply:
For a second I was confused thinking this _is_ the function definition and declaration, as it's where the class is defined.  Now I see that it inherits from another class and understand why void is redundant here.  I think `override` would help make this more obvious and I may have realized.

Reply:
For closure, there was a warning in the editor for missing override that I didn't notice while trying to resolve the ton of other messages.  Proves that a 4th pass at the warnings would've helped